lib/utils/password_manager.dart

```diff
@@ -0,0 +1,53 @@
+import 'dart:convert';
+
+import 'package:reef_mobile_app/model/ReefAppState.dart';
+import 'package:reef_mobile_app/model/StorageKey.dart';
+import 'package:reef_mobile_app/model/account/stored_account.dart';
+import 'package:reef_mobile_app/service/JsApiService.dart';
+
+class PasswordManager {
+  static Future<bool> checkIfPassword() async {
+    try {
+      final password = await ReefAppState.instance.storageCtrl
+          .getValue(StorageKey.password.name);
+      if (password == null) return false;
+      return true;
+    } catch (e) {
+      return false;
+    }
+  }
+
+  static void updateAccountPassword(
+      StoredAccount account, String newPass) async {
+    // to only split from the first occurence considering that password might contain '+' in it    
+    int delimiterIdx = account.mnemonic.indexOf("+");
+    List<String> accountElems = [
```

Comment:
better to use split

Reply:
this is used in several places so create a method

Reply:
- if we use split and the user has password which contains + , for example "anukul+pandey" is password then the array we will get will be [addr,anukul,pandey] , where as the password is anukul+pandey , so using delimiterIdx is better or else we will need to iterate the resulting array to concat password .

- yea will create a method for it